src/lib/media-import/media-file-import.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @flow
+ * @format
+ */
+
+/**
+ * Internal dependencies
+ */
+import { GB_IN_BYTES, MB_IN_BYTES } from 'lib/constants/file-size';
+
+export const MEDIA_IMPORT_FILE_SIZE_LIMIT = 30 * GB_IN_BYTES;
+
+export interface AppForMediaImport {
+	id: Number;
+	environments: Array<any>;
+	name: string;
+	organization: Object;
+	type: string;
+}
+
+export function currentUserCanImportForApp( app: AppForMediaImport ): boolean {
+	// TODO: implement
```

Comment:
Are we missing some logic here, or are we releasing without this?

Reply:
this was copied over from `db-file-import.js` , I think it save to disregard